### DISCUSSION TITLE
Using spatial bookmarks in QField

### DISCUSSION THREAD

**Discussion by gdsrg** [opengisch/QField#2722] [Q&A]
I set a couple spatial bookmarks in my QGIS project. They show up correctly in QField search results and also in the map. 
However sometimes the bookmark symbol is in the way. Is there a way to switch bookmarks to invisible inside QField or configure them in any way beforehand so they can be turned off?

**Comment by nirvn** [comment]
@gdsrg , a new setting to toggle bookmarks on/off will be available in QField 2.1: https://github.com/opengisch/QField/pull/2745#issuecomment-1098998615
